modules/concepts/hooks/list-of-hooks.md

```diff
@@ -2806,4 +2862,131 @@ actionAfterCreate&lt;FormName>FormHandler
     ]
     ```
 
+actionFilterDeliveryOptionList
+: 
+Available since: {{< minver v="8.0.0" >}}
+Located in: /classes/Cart.php
+
+    Parameters:
+    ```php
+    <?php
+    [
+        'delivery_option_list' => (array) &$delivery_option_list,
+    ]
+    ```
+
+actionValidateOrderAfter
+: 
+Available since: {{< minver v="8.0.0" >}}
+Located in: /classes/PaymentModule.php
+
+    Parameters:
+    ```php
+    <?php
+    [
+        'cart' => (Cart|null) $contextCart,
+        'order' => (Order|null) $order,
+        'orders' => (array) $orderList,
+        'customer' => (Customer) $contextCustomer,
+        'currency' => (Currency) $contextCurrency,
+        'orderStatus' => (OrderState) $orderState,
+    ]
+    ```
+
+actionOrderStatusUpdate
+: 
+Available since: {{< minver v="8.0.0" >}}
```

Comment:
```suggestion
Available since: {{< minver v="1.5.5" >}}
```

Reply:
oh this hook was defined correctly in docs already. Removed my change. :+1: